README.md

```diff
@@ -6,10 +6,18 @@
 
 •go out alot
 
+answer-one
 •watch old history documentaries
 
 # proflile links
 this is the first time learning about github and i,m excited to learn something new.
 
 # tech skills
 none at the moment 
+=======
+
+=====
+# about-me 
+my name is joshua deguerre im 19 years old and i'm very new to coding 
+
+master
```

Comment:
These tags are provided by git when we have a conflict so that we know what branch each of the changes came from. We'd want to remove them during conflict resolution. When we are done resolving conflicts the file should only have the contents we want as our final document. In code, leaving these in will most likely break the code since they aren't meant to be part of the code but were just temp tags provided by git.